modules/installation-aws-config-yaml.adoc

```diff
@@ -150,15 +150,19 @@ platform:
   aws:
 ifndef::gov,china,secret[]
     region: us-west-2 <1>
+    propagateUserTags: true <3>
```

Comment:
The description in call out <3> under this code block is as follows:

> <3> If you do not provide these parameters and values, the installation program provides the default value.

I noticed the call out applies to multiple different lines, so it can't be updated to provide more specific details about the `propagateUserTags` parameter. You could add a second call out for this line that provides some details about the parameter, possibly matching the description that you are adding in line 761. This will provide a quick reference for the reader directly under the code block, in addition to the details provided in the table earlier in the page.

Reply:
I tried that but with the heavy usage of conditionals and the numbering change for each aws assembly that this module is is used in, I discussed with @mjpytlak and the ideal placeholder would be on line 761.

Reply:
Hi @pneedle-rh. @Amrita42 and I discussed this ahead of her updates. This module is in need of some refactoring (as are all of the sample files, IMO). While the use of conditionals permits reuse, it is so heavily conditioned that it has become extremely fragile. Further, we do have precedence for not describing every last parameter in the sample file. For the time being, the description of `propogateUserTags` is good to live in Installation configuration parameters > Optional AWS configuration parameters. Stephanie and I are aware of the state of the sample install configuration file and are prioritizing various levels of refactoring.